cmd/kube-proxy/app/server.go

```diff
@@ -1111,6 +1122,8 @@ func getNodeIPs(ctx context.Context, client clientset.Interface, name string) []
 	})
 	if err == nil {
 		logger.Info("Successfully retrieved node IP(s)", "IPs", nodeIPs)
+	} else {
+		logger.Info("Error to retrieve node IP(s)", "error", err)
```

Comment:
Whereas this is actually an error. But `err` is just going to be `timed out`.
It might be nicer to do the "preserve the last error returned from GetNodeHostIPs and only log it if the loop eventually times out rather than logging it each time" thing.